GNUmakefile

```diff
@@ -137,6 +141,7 @@ tools:
 	cd tools && go install github.com/terraform-linters/tflint
 	cd tools && go install github.com/pavius/impi/cmd/impi
 	cd tools && go install github.com/hashicorp/go-changelog/cmd/changelog-build
+	cd tools && go install github.com/rhysd/actionlint/cmd/actionlint@latest
```

Comment:
For the other tools, we're locking the version using the `go.mod` file in `tools/`. If we're always installing `@latest`, we don't need to change directories first. However, we'd then lose the benefit of working with a stable version.

Reply:
ooo yess good point! will move that over to go.mod 👍